11-multi-track/bb_views_part1/readme.md

```diff
@@ -172,14 +179,14 @@ And the in our index.html:
 
 ### Exercise: Define a Handlebars Template (10 minutes)
 
-1. Define a handlebars template for our grumble. (See the sample HTML below)
+1. Define a Handlebars template for our Grumble. See the sample HTML below.  
```

Comment:
Indicate that this needs to be placed as a separate script in `index.html` with `id=grumbleTemplate`.